packages/commonwealth/test/devnet/cosmos/cosmosGovNotifGenerator.spec.ts

```diff
@@ -50,12 +56,34 @@ async function enableChains(chains: string[]) {
     },
   });
 
-  for (const id of possibleChains) {
-    if (chains.includes(id)) {
+  for (const id of possibleCommunities) {
+    if (communities.includes(id)) {
+      const [chainNode] = await models.ChainNode.findOrCreate({
+        where: {
+          cosmos_chain_id: id,
+          balance_type: BalanceType.Cosmos,
+          name: `${id}Node`,
+          url: `http://localhost:8080/cosmosAPI/${id}`,
+          alt_wallet_url: `http://localhost:8080/cosmosAPI/v1/${id}`,
+        },
+      });
+      const [community] = await models.Community.findOrCreate({
+        where: {
+          id,
+        },
+        defaults: {
+          name: id,
+          network: ChainNetwork.Osmosis,
+          type: ChainType.Chain,
+          base: ChainBase.CosmosSDK,
+          default_symbol: 'STAKE',
+          chain_node_id: chainNode.id,
+        },
+      });
```

Comment:
@Rotorsoft I got this closer to the data we need, but something is still not right. @timolegros is more familiar with these particular tests, so maybe he can help.